sklearn/linear_model/base.py

```diff
@@ -397,7 +398,8 @@ def fit(self, X, y):
         return self
 
 
-def _pre_fit(X, y, Xy, precompute, normalize, fit_intercept, copy):
+def _pre_fit(X, y, Xy, precompute, normalize, fit_intercept, copy,
+             aux_order=None):
```

Comment:
I don't understand what "aux" stands for. Is it an acronym? A more explicit is probably required.